src/utilities.js

```diff
@@ -157,3 +157,20 @@ export function createMelFilterBank(numFilters, sampleRate, bufferSize) {
 
   return filterBank;
 }
+
+export function frame(buffer, frameLength, hopLength) {
+  if (buffer.length < frameLength) {
+    throw new Error('Buffer is too short for frame length');
+  }
+  if (hopLength < 1) {
+    throw new Error('Hop length cannot be less that 1');
+  }
+  if (frameLength < 1) {
+    throw new Error('Frame length cannot be less that 1');
+  }
+
+  var numFrames = 1 + Math.floor((buffer.length - frameLength) / hopLength);
```

Comment:
this could be a `const`

Reply:
Considering everything else uses `var` or `let` probably keep it as `var` for now.
We should probably add some sort of linting and bump everything to `ES2015+` soon however.

Reply:
Ah, didn't see this was already amended, not a problem - I'll create an issue to lint and update the code